package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "leveldown",
-  "description": "A Node.js LevelDB binding, primary backend for LevelUP",
+  "description": "A node.js LevelDB binding, primary backend for levelup",
```

Comment:
Do we really need "Node.js" in the description of an npm package?

Reply:
Not really. But maybe it should be the same as the short descr in the readme.

Reply:
Alright